Neos.Media/Classes/Domain/Service/ThumbnailService.php

```diff
@@ -92,11 +92,6 @@ public function getThumbnail(AssetInterface $asset, ThumbnailConfiguration $conf
             if ($asset->getWidth() === null && $asset->getHeight() === null) {
                 return $asset;
             }
-            $maximumWidth = ($configuration->getMaximumWidth() > $asset->getWidth()) ? $asset->getWidth() : $configuration->getMaximumWidth();
-            $maximumHeight = ($configuration->getMaximumHeight() > $asset->getHeight()) ? $asset->getHeight() : $configuration->getMaximumHeight();
-            if ($configuration->isUpScalingAllowed() === false && $maximumWidth === $asset->getWidth() && $maximumHeight === $asset->getHeight()) {
-                return $asset;
-            }
```

Comment:
In case we decide to remove those lines, the doc block needs to be updated https://github.com/neos/neos-development-collection/pull/1480/files#diff-16e8cf609f0f20b6ffe12ce13b5f5679L79